tests/test_identity_map_client.py

```diff
@@ -166,6 +171,20 @@ def assert_unmapped(self, response, reason, dii):
         mapped_identity = response.mapped_identities.get(dii)
         self.assertIsNone(mapped_identity)
 
+    def test_identity_buckets(self):
+        response = self.identity_map_client.get_identity_buckets(datetime.datetime.now() - datetime.timedelta(days=2))
+        self.assertTrue(len(response.buckets) > 0)
+        self.assertTrue(response.is_success)
+
+    def test_identity_buckets_empty_response(self):
+        response = self.identity_map_client.get_identity_buckets(datetime.datetime.now() + datetime.timedelta(days=1))
+        self.assertTrue(len(response.buckets) == 0)
+        self.assertTrue(response.is_success)
+
+    def test_identity_buckets_invalid_timestamp(self):
```

Comment:
I think this would be better as a unit test.

Reply:
I added more test cases in `test_identity_buckets_invalid_timestamp`